views/models/UserConfigModel/index.js

```diff
@@ -0,0 +1,56 @@
+import colorbrewer from 'colorbrewer'
+
+const UserConfigModel = {
+  configColorOptions: () => {
+    return Object.keys(colorbrewer)
+      .map((key) => colorbrewer[key])
+      .filter((_, i) => i !== 0)
+      .map((palettes) => Object.values(palettes))
+      .flatMap((palette) => palette)
+      .map((palette, i) => ({ label: palette, value: i }))
```

Comment:
I wonder if we can optimize this down one (or two) `map` functions?

Reply:
This also feels like it could move to a helper; it doesn't look like it relies on any data related to the UserConfigModel?

Reply:
Thanks for this. 
[5ba7227](https://github.com/AMP-SCZ/dpdash/pull/332/commits/5ba722717d57e6c34b0a2c188110d767823c7695)

Reply:
[65869b2](https://github.com/AMP-SCZ/dpdash/pull/332/commits/65869b277331752ca789dcb0ebaaf8ba126db2ee)